modules/person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    rentals << rental
+  end
+
```

Comment:
- [ ] Kindly make modification to the method and make sure rental is added as a new instance. [Link to the example in a previous lesson:]( https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)

### Sample code
```ruby
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```